src/terminal/osc.zig

```diff
@@ -1634,6 +1676,61 @@ test "OSC: set palette color" {
     try testing.expectEqualStrings(cmd.set_color.value, "rgb:aa/bb/cc");
 }
 
+test "OSC: conemu sleep" {
+    const testing = std.testing;
+
+    var p: Parser = .{};
+
+    const input = "9;1;420";
+    for (input) |ch| p.next(ch);
+
+    const cmd = p.end('\x1b').?;
+
+    try testing.expect(cmd == .sleep);
+    try testing.expectEqual(420, cmd.sleep.duration_ms);
+}
+
+test "OSC: conemu sleep with no value default to 100ms" {
+    const testing = std.testing;
+
+    var p: Parser = .{};
+
+    const input = "9;1;";
+    for (input) |ch| p.next(ch);
+
+    const cmd = p.end('\x1b').?;
+
+    try testing.expect(cmd == .sleep);
+    try testing.expectEqual(100, cmd.sleep.duration_ms);
+}
+
+test "OSC: conemu sleep cannot exceed 10000ms" {
+    const testing = std.testing;
+
+    var p: Parser = .{};
+
+    const input = "9;1;12345";
+    for (input) |ch| p.next(ch);
+
+    const cmd = p.end('\x1b').?;
+
+    try testing.expect(cmd == .sleep);
+    try testing.expectEqual(10000, cmd.sleep.duration_ms);
+}
+
+test "OSC: conemu sleep invalid input" {
+    const testing = std.testing;
+
+    var p: Parser = .{};
+
+    const input = "9;1;foo";
+    for (input) |ch| p.next(ch);
+
+    const cmd = p.end('\x1b');
+
+    try testing.expect(cmd == null);
+}
```

Comment:
Note to myself: well that should default to `100` to fully match conemu's behaviou

Reply:
addressed in 9d9fa60ece3736001be39ec2ac80b0e6c4b44f17